qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -1632,12 +1631,97 @@ All resulting points are sorted by ID. To query the next page it is necessary to
 For convenience, this ID is also returned in the field `next_page_offset`.
 If the value of the `next_page_offset` field is `null` - the last page is reached.
 
-<!-- 
-Python client:
+### Order points by a payload key
+
+_Available as of v1.8_
+
+When using the [`scroll`](#scroll-points) API, it is possible to sort the results by the contents of a payload field. This is useful when you want to retrieve points in a specific order, for example, by a timestamp, for chronological order.
+
+<aside role="status">Without an appropriate index, custom ordering would be a very expensive operation. For this reason, in Qdrant, it is a requirement to have an index capable of <a href=/documentation/concepts/indexing/#payload-index target="_blank">Range filtering conditions</a> on the field you want to `order_by`. Once the index is set up, these ordering operations can be really cheap.</aside>
+
+```http
+POST /collections/{collection_name}/points/scroll
+{
+    "limit": 15,
+    "order_by": "timestamp", // <-- this!
+}
+```
 
 ```python
+client.scroll(
+    collection_name="{collection_name}",
+    limit=15,
+    order_by="timestamp", # <-- this!
+)
 ```
- -->
+
+```typescript
+client.scroll("{collection_name}", {
+  limit: 15,
+  order_by: "timestamp", // <-- this!
+});
+```
+
+```rust
+use qdrant_client::qdrant::{Condition, Filter, ScrollPoints, OrderBy};
+
+client
+    .scroll(&ScrollPoints {
+        collection_name: "{collection_name}".to_string(),
+        limit: Some(15),
+        order_by: Some(OrderBy {
+            key: "timestamp".to_string(),  // <-- this!
+            ..Default::default(),
+        }),
+        ..Default::default()
+    })
+    .await?;
+```
+
+The `order_by` `key` parameter specifies the payload key to order the results by, but there are also other fields that can be set to control the ordering, like `direction` and `start_from`:
```

Comment:
```suggestion
The `order_by` `key` parameter is one way to order the results. You can substitute other parameters for `key` such as `direction` and `start_from`:
```

Reply:
I don't feel strongly about this, will approve whether you commit this suggestion or not.

Reply:
Yeah, I think the suggestion _suggests_ that the `key` param is optional, but it is not. What I wanted to convey is that you can specify these other extra parameters along with the `key`

Reply:
Ah thank you for explaining. Then maybe:

```suggestion
Use the `order_by` `key` parameter to specify the payload key. You can then add other fields to control the ordering, like `direction` and `start_from`:
```